css/mediaqueries/META.yml

```diff
@@ -911,16 +911,12 @@ links:
       results:
         - test: device-aspect-ratio-004.html
         - test: mq-calc-008.html
-        - test: mq-gamut-002.html
-        - test: mq-gamut-004.html
```

Comment:
It looks like color-gamut is part of MQ 4 as well: https://drafts.csswg.org/mediaqueries-4/#color-gamut

I got confused because it's both part of MQ 5 and 4:

https://drafts.csswg.org/mediaqueries-5/#color-gamut
https://drafts.csswg.org/mediaqueries-4/#color-gamut

Reply:
ok so I think I got it, video-color-gamut is MQ5 and color-gamut is MQ4

Reply:
Thanks!